tools/mason/MasonPublish.chpl

```diff
@@ -459,6 +463,17 @@ proc check(username : string, path : string, trueIfLocal : bool, ci : bool) thro
     writeln(spacer);
   }
 
+  if package {
+    writeln('Checking for fields in manifest file:');
+    if masonTomlFileCheck(projectCheckHome) {
+      writeln('   All fields present in manifest file, can be published to a registry. (PASSED)');
+    } else {
+      writeln('   Missing fields in manifest file (Mason.toml). (FAILED)');
```

Comment:
I know it would not be trivial to add, but I think printing the fields missing would really help users debug why their package is failing the check.